portfolio/src/main/webapp/index.html

```diff
@@ -10,60 +10,76 @@
 
   <body>
     <!--Home Initial View-->   
-    <header id="home" class="bg">
+    <header id="home" class="home-background">
       <div id="nav">
         <ul>
-          <li><a href="#gallery">Gallery</a></li>
-          <li><a href="#projects">Projects</a></li>
-          <li><a href="#timeline">Timeline</a></li>
-          <li><a href="#about">About Me</a></li>
-          <li><a href="#home">Home</a></li>
+          <li><a href="gallery.html">Gallery</a></li>
+          <li><a href="index.html#to-do">To-Do's</a></li>
+          <li><a href="index.html#projects">Projects</a></li>
+          <li><a href="index.html#timeline">Timeline</a></li>
+          <li><a href="index.html#about">About Me</a></li>
+          <li><a href="index.html#home">Home</a></li>
         </ul>  
       </div>
       <div class="absolute-center">
         <h1>Sherry Shi</h1>
         <p>Pictured: Port Washington Sunset | pc: Google Images</p>
-        <p>Click here to get a random fact about me:</p>
-        <button onclick="genRandomFact()">Fact</button>
-        <p id="fact-container"></p>
       </div>
     </header>
 
     <!-- About Section --> 
     <section id="about" class="dark">
       <div class="container">
+        <!--Intro Paragraph-->
         <h3 class="blue-text">About Me</h3>
         <div class="wrapper bio">
           <img class="pro-pic" src="/images/ryan.jpg" alt = "My Profile Picture">
           <div class="intro">
             <p>My name is Sherry. I'm a rising junior at the University of 
                 Pennsylvania currently pursuing a dual degree in Computer Science
                 from Engineering and business from Wharton.</p>
+            <button onclick="genRandomFact()">Random Fact About Me</button>
+            <p id="fact-container"></p>
           </div>
         </div>
+        <!--Three-columned Profile-->
         <div class="row wrapper profile">
-          <div class="column half">
-            <h4 class="blue-text">Basic Info</h4>
-            <b>Date of Birth:</b> 
-            <p>04/13/2001</p>
-            <b>Hometown:</b> 
-            <p>Port Washington, NY</p>
-            <b>Languages:</b>
-            <p>English \& Chinese</p>
-          </div>
-          <div class="column half">
-            <h4 class="blue-text">Interests & Hobbies</h4>
-            <b>Music</b> 
+          <div class="column third">
+            <div class="profile-card">
+              <h4 class="blue-text">Basic Info</h4>
+              <b>Date of Birth:</b> 
+              <p>04/13/2001</p>
+              <b>Hometown:</b> 
+              <p>Port Washington, NY</p>
+              <b>Languages:</b>
```

Comment:
According to the HTML5 specification, the <b> tag should be used as a LAST resort when no other tag is more appropriate. `<strong>` would be better or if you wanted to make sure each item is on a new line, you can use a div and change the font weight attribute.

Reply:
okay! changed